mcf-browser/src/back-end/parse-tmcf.ts

```diff
@@ -174,6 +175,111 @@ class ParseTmcf {
     return mcfLines.join('\n');
   }
 
+  /**
+   * Convert a single row to a string representing facets
+   * and its corresponding value and date
+   * @param {string} template The string representation of a tmcf file.
+   * @param {Object} row The json representation of the csv row.
```

Comment:
I created a type for most of the other instances but in this specific line of code, it's impossible to get any more specific because the object properties are named after the column headers of the csv which changes per csv file.

Reply:
In this case, you can define the type as { [header:string]: string } so we know what the keys represent and what the type should be for the values